CBinding/ProjectPad/SystemFolderNodeBuilder.cs

```diff
@@ -72,11 +72,6 @@ public override object GetParentObject (object dataObject)
 			var folder = (SystemFolder)dataObject;
 			return new SystemFolder (folder.Path.ParentDirectory, folder.ParentWorkspaceObject, false);
 		}
-
-		public override string GetFolderPath (object dataObject)
-		{
-			return ((SystemFolder)dataObject).Path;
-		}
 	}
```

Comment:
@mhutch ,this works fine , can you please check this ?
Apart from these changes, we need this to be public - https://github.com/mono/monodevelop/blob/master/main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.Gui.Components/ExtensibleTreeView.cs#L601